### ISSUE TITLE
Inconsistent Iris dataset

### ISSUE THREAD

**Issue by 53X** [Jeetu95/Decision-Tree#1]
The iris dataset you have used is not consistent with the original one.

**Comment by sdsubhajitdas** [comment]
Yes i agree with you the iris dataset is not consistent. This code was actually written for my college project so the output format is a bit custom sort of and this code was actually designed to go with the other two dataset . The tested this with the iris dataset just for a bit fun.
